pkg/driverbuilder/builder/templates/alinux.sh

```diff
@@ -28,10 +28,7 @@ rm -Rf /tmp/module-download
 mkdir -p /tmp/module-download
 
 curl --silent -SL {{ .ModuleDownloadURL }} | tar -xzf - -C /tmp/module-download
-mv /tmp/module-download/*/driver/* {{ .DriverBuildDir }}
-
-cp /driverkit/module-Makefile {{ .DriverBuildDir }}/Makefile
-bash /driverkit/fill-driver-config.sh {{ .DriverBuildDir }}
+mv /tmp/module-download/*/* {{ .DriverBuildDir }}
```

Comment:
Scripts are a bit different now: 
* we move all the stuff under the download libs repo to`{{ .DriverBuildDir }}` now, not only the `driver` subfolder
* we run `cmake` with lots of things disabled. Unfortunately, given that libs only [recently](https://github.com/falcosecurity/libs/pull/1325) gained the ability to build the bpf probe in `MINIMAL_BUILD` mode, we need to configure the project in full mode, and this requires `git` because `grpc` and `protobuf` deps need to be fetched/cloned.
* Then, building is just a matter of calling `make driver` and `make bpf` with the correct `CC` and `KERNELDIR`.

Reply:
I actually really like how much this all simplifies the code, and I think the trade offs are worth it.